src/GraphQL.SystemTextJson/ExecutionResultJsonConverter.cs

```diff
@@ -123,8 +161,20 @@ private static void WriteValue(Utf8JsonWriter writer, object value, JsonSerializ
 
                     break;
                 }
+                case object[] list:
```

Comment:
Can be merged in one case with `case List<object> list:`  -> ` case IList list:` + `foreach` -> `for` but I did not.